### PR TITLE
[zh] Add 10 .md localizations

### PR DESCRIPTION
**Pull request by Jacob953**
**Done**

- [x] Add cluster.md localization to zh.
- [x] Add cloud_native_apps.md localization to zh.
- [x] Add cloud_native_security.md localization to zh.
- [x] Add contianer_image.md localization to zh.
- [x] Add containization.md localization to zh.
- [x] Add contianer_as_a_server.md localization to zh.
- [x] Add kubernetes.md localization to zh.
- [x] Fix typos of cloud_native_tech.md
- [x] Add countinuous_integration.md localization to zh.
- [x] Add countinuous_delivery.md localization to zh.

### CODE REVIEW

**Comment by hanyuancheung** [comment]
Good work! This PR should be reviewed after https://github.com/cncf/glossary/pull/489. Before that, don't forget to rebase it.

File: content/zh/cluster.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+category: 概念
+---
+
+## 是什么
+
+集群是一组计算机或应用程序，它们为一个共同的目标一起工作。在云原生计算的背景下，这个术语最常被应用于 Kubernetes。Kubernetes 集群是一组服务（或工作负载），它们在各自的容器中运行，通常在不同的机器上。所有这些 [容器化](/containerization/) 服务的集合，通过网络连接，代表一个集群。
+
+## 强调的问题
+
+在单台计算机上运行的软件会出现单点故障——如果这台计算机崩溃了，或者有人不小心拔掉了电源线，那么一些关键的业务系统就可能被切断。这就是为什么现代软件通常被构建为 [分布式应用](/distributed_apps/)，以集群的形式组合在一起。
```
Comment (Submarinee) [inline content/zh/cluster.md:13]: 「那么一些关键的业务系统就可能被切断」的原文是「some business-critical system may be taken offline.」
taken offline翻译为下线或者脱机是不是更好？
仅为个人意见。

[the remark this answers is not in the document]
Comment (hanyuancheung) [reply content/zh/cluster.md:13]: "下线" works for me!

[the remark this answers is not in the document]
Comment (Jacob953) [reply content/zh/cluster.md:13]: I'll take this advice. Thx!

**Comment by Submarinee** [comment]
LGTM.

**Comment by Jacob953** [comment]
I just figured out some conflict here by my mistakes.

container_image.md #590 and continuous_delivery.md #591 existed.

**Review by hanyuancheung — APPROVED** [review]
Brilliant! LGTM. Thanks for your contribution.

The other new items should be split into different PRs, this one is forgiven.

**Review by Rocksnake — APPROVED** [review]
Thanks~

**Pull request merged**